cmd/promtool/main.go

```diff
@@ -291,8 +291,6 @@ func main() {
 	promQLLabelsDeleteQuery := promQLLabelsDeleteCmd.Arg("query", "PromQL query.").Required().String()
 	promQLLabelsDeleteName := promQLLabelsDeleteCmd.Arg("name", "Name of the label to delete.").Required().String()
 
-	featureList := app.Flag("enable-feature", "Comma separated feature names to enable (only PromQL related and no-default-scrape-port). See https://prometheus.io/docs/prometheus/latest/feature_flags/ for the options and more details.").Default("").Strings()
```

Comment:
Question for a maintainer, maybe @beorn7 : Should this flag be dropped or should it be kept, but with no feature to enable for now?

Reply:
I would remove it entirely. It's easy to bring back, but having this command line flag without any feature flags sounds weird. The other side of the argument would be that it is almost certain that we'll have feature flags in promtool, too, before we release v4. @dgl should make the call here.

Reply:
Yet another argument is to not needlessly break users that still pass in the feature flag. It shouldn't be a problem to activate a feature flag that is now default.

This might convince me to keep the `--enable-feature` flag in `promtool`, just without any possible option that would do anything (as it was done in #14958, the PR that weirdly was created mirroring this PR).

Reply:
That's a good reason, IMO. I'll add a dummy flag back in.